Framework/Algorithms/src/FindPeaks.cpp

```diff
@@ -1087,7 +1087,7 @@ int FindPeaks::findPeakBackground(const MatrixWorkspace_sptr &input,
         "No 7th column for use FindPeakBackground result or not. ");
 
   if (peaklisttablews->rowCount() > 0) {
-    int fitresult = peaklisttablews->Int(0, 6);
+    fitresult = peaklisttablews->Int(0, 6);
```

Comment:
That apparently innocent line is a can of worms: https://github.com/mantidproject/mantid/pull/13667, https://github.com/mantidproject/mantid/issues/13950 (from a coverity issue that caught this).